AnimationProfiles/Readme.md

```diff
@@ -15,5 +15,31 @@ Each folder contains the **animation profiles** along with domain and problem fi
 11. [Freecell](http://editor.planning.domains/#read_session=q8Rh3Y7sSE)
 12. [Depot](http://editor.planning.domains/#read_session=MFmtS8g0ar)
 13. [FlowFree](http://editor.planning.domains/#read_session=dbnH15yyMF)
-14. [TPP] (http://editor.planning.domains/#read_session=mz2BJxLOb5)
-15. [Gripper] (http://editor.planning.domains/#read_session=jKknyAIR29)
+14. [Gripper](http://editor.planning.domains/#read_session=jKknyAIR29)
+15. [Sokuban](https://editor.planning.domains/#read_session=qSABTuifk3)
+17. [Zenotravel](https://editor.planning.domains/#read_session=sX9kra1FQm)
+18. [Floortile](https://editor.planning.domains/#read_session=qUo9dmdKgG)
+19. [Hiking](http://editor.planning.domains/#read_session=LwNY8sppTZ)
+20. [Nurikabe](https://editor.planning.domains/#read_session=iKykgU5Ozm)
+21. [Peg](https://editor.planning.domains/#read_session=9xfAMlLMDg)
+22. [Snake](https://editor.planning.domains/#read_session=T683ydMmDw)
+
+
+
+## Quick load samples
+
+You can access [Planimation Visualisation](http://47.236.182.248:8080/problem) to load these demo quickly. These demo all connect to this repository.
```

Comment:
Thanks for the new landing page to upload problems. I tested the icon at the left corner, and it doesn't seem to be working. 

It would be great to troubleshoot it and to push the changes in the front-end and backend so we can deploy it in the server that is hosting the current landing page https://planimation.planning.domains/

![Screenshot 2024-07-24 at 2 24 04 PM](https://github.com/user-attachments/assets/73a81852-8082-469b-b6a4-5b5d9158fafd)

Reply:
Hi, Nir, sorry for the late reply, I am busy for finding a job recently.
In the frontend code, I just catch the files which obey some rules, I put the note into readme.md of Animation profiles.
- Keep your directory name clear and easy to know what it is.
- The domain files must have `domain` in its name. e.g. `puzzle_domain.pddl`
- The problem files must have `problem`in its name. e.g. `puzzle_problem1.pddl`
- The ap files must have `ap` in its name. e.g. `puzzle_ap.pddl`
Jian Zhang already change the animation profiles into [branch](https://github.com/planimation/documentation/tree/Team_Koala_AP_Files/AnimationProfiles) . And Yue Ying Changed the other team's animation profiles in [PR22](https://github.com/planimation/documentation/pull/22/files).

Or you need other changes, please email me or just comment me, I will do it on the weekend or next week.